docs/en/contributing/creating-packages.md

```diff
@@ -0,0 +1,136 @@
+
+When a testing application is needed, it should be created in an `app` directory at the root of the package.
+
+![Example of tracked folder structure with fast-package at the top and __tests__, app and src inside.](https://res.cloudinary.com/fast-dna/image/upload/v1546297640/creating_packages_tracked_folders.webp "")
+*Example structure for tracked folders.*
+
+### Untracked folders
+
+Untracked folders are folders included in the `.gitignore` and generated by an automated process.
+
+These may include:
+
+* A `node_modules` folder for npm dependencies -- generated by the `npm i` command.
+* A `dist` folder for the exported code -- generated by compiling TypeScript and specified by the `tsconfig.json`.
+* A `www` folder for exported compiled applications if creating an `app` folder -- generated by `webpack` and specified by the `webpack.config.js`.
+* A `coverage` folder for test coverage reporting.
+
+## package.json
+
+All packages use npm and are published to the [npm registry](https://www.npmjs.com/).
+
+### "sideEffects": false
+
+All packages are expected to be side-effect free. Adding a `"sideEffects": false` flag to the package.json indicates to tools like webpack that the package's modules have no side effects (on evaluation) and only expose exports. This allows these tools to optimize re-exports through tree-shaking and dead code removal.
+
+### Scripts
+
+Unless specified, scripts are defined in the `package.json` scripts property so they can be run with `npm run [script-name]`.
+
+The following scripts are required:
+
+* `npm run build` builds the exported package.
+* `npm run clean:dist` cleans the distribution folder.
+* `npm run prepare` cleans the distribution folder and builds the exported package.
+* `npm run prettier` ensures code is formatted correctly (runs on a global repository githook).
+* `npm run prettier:diff` provides a hook to ensure code formatting matches expectations on the build server.
+* `npm run test` runs all tests.
+* `npm run tslint` runs tslint.
+
+Additional scripts and common use cases include:
+
+* `npm start` starts the webpack-dev-server.
+* `npm run tslint:fix` runs tslint with the fix option.
+* `npm run watch` watches the files and builds the exported package when making changes to the source files.
+
+## .npmignore
+
+The `.npmignore` file removes:
+
+* All testinging files; usually by targetting the `__tests__`, `*.spec.*` and `*.test.*` patterns.
+* All unminified code; usually by targetting the `src` folder.
+* The `babel.config.js` if it has been added to the package.
+
+## Webpack
+
+A Webpack application can be created using `webpack` and `webpack-dev-server` for testing and/or documentation purposes. If it is necessary to publish a webpack application that serves as a documentation site to a live site, a production webpack build should be made available.
```

Comment:
Webpack is something lowercase and sometimes capitalized - I think we should stick to one or the other